packages/lib/server/getLuckyUser.ts

```diff
@@ -596,6 +655,11 @@ async function fetchAllDataNeededForCalculations<
 
   const interval = getLuckyUserParams.eventType.team?.rrResetInterval ?? RRResetInterval.MONTH;
 
+  const rrTimestampBasis =
+    eventType.isRRWeightsEnabled && getLuckyUserParams.eventType.team?.rrTimestampBasis
```

Comment:
Timestamp basis is only for weighted round robin. If it's not weighted then we pick least recently booked user (which always uses the Created At timestamp)

I also changed the setting description (see updated screenshot in PR description)